pkgs/tools/typesetting/pdfchain/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchurl
+, autoconf, gtkmm3, glib, pdftk, pkg-config, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pdfchain";
+  version = "0.4.4.2";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/${pname}/${pname}-${version}/${pname}-${version}.tar.gz";
+    sha256 = "sha256-Hu4Pk9voyc75+f5OwKEOCkXKjN5nzWzv+izmyEN1Lz0=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config wrapGAppsHook autoconf
+  ];
+
+  buildInputs = [
+    gtkmm3 pdftk glib
+  ];
+
+  patches = [
+    ./pdfchain-fix-crash-on-startup.patch
+    ./pdfchain-fix-desktop-file.patch
+    ./pdfchain-fix-spelling.patch
```

Comment:
Looks like these patches come from debian. Can please you fetch them with fetchpatch instead?

Reply:
Will do